tensorflow_lite_support/tools/ci_build/builds/build_ios_framework.sh

```diff
@@ -58,140 +60,51 @@ if [[ -z "${DEST_DIR+x}" || "${DEST_DIR}" == ${TFLS_ROOT_DIR}* ]]; then
   exit 1
 fi
 
-
-# Builds the C API framework for the specified framework for iOS.
-function build_c_api_framework {
-  "${BAZEL}" build -c opt --config=ios_fat \
-    //tensorflow_lite_support/ios:$1C_framework
+# get_output_file_path takes one bazel target label as an argument, and prints
+# the path of the first output file of the specified target.
+function get_output_file_path {
+  local STARLARK_FILE="${OUTPUT_TMPDIR}/print_output_file.starlark"
+  cat > "${STARLARK_FILE}" << EOF
+def format(target):
+  return target.files.to_list()[0].path
+EOF
+  bazel cquery --config=ios_fat $1 \
+    --output=starlark --starlark:file="${STARLARK_FILE}" 2> /dev/null
 }
 
-function create_framework_archive {
-  TARGET_FRAMEWORK_NAME="$1"
-  C_API_FRAMEWORK_NAME="$1C"
+function build_ios_api_framework {
+  local TARGET="//tensorflow_lite_support/ios"
```

Comment:
nit: `TARGET` -> `TARGET_PREFIX`

Reply:
@priankakariatyml You seem to have missed this comment.

Reply:
Fixed